### PR TITLE
don't use request.REQUEST

### PR DESCRIPTION
**Pull request by ryanmrubin**
Since it's not available in Django 1.9.

### CODE REVIEW

**Comment by eykd** [comment]
Seriously? They got rid of `request.REQUEST`? But WHYYY?

**Comment by ryanmrubin** [comment]
Not 100% sure. But they definitely are.

**Comment by ryanmrubin** [comment]
I can trigger the "force-pushed branch" thing from here now.

**Comment by eykd** [comment]
WWWHYYYYYYYYY?!

**Comment by eykd** [comment]
Apparently, using `request.REQUEST` is BAD because PHP: https://code.djangoproject.com/ticket/18659

**Comment by dwickwire** [comment]
Fascinating.

File: cas_consumer/views.py [hunk 1]
```diff
@@ -30,8 +30,10 @@ def login(request, redirect_field_name=REDIRECT_FIELD_NAME):
 
     """
     ticket = request.GET.get(settings.CAS_TICKET_LABEL, None)
-    redirect_to = request.REQUEST.get(redirect_field_name,
-                                      request.session.pop('login_redirect_to', ''))
+    get_or_post_params = request.GET.copy()
+    get_or_post_params.update(request.POST)
```
Comment (eykd) [inline cas_consumer/views.py]: We should probably just check `request.method` instead of doing the merging.

Comment (ryanmrubin) [inline cas_consumer/views.py]: request.get(request.method, {}) would work, so long as there's never anything but GET or POST. Do we want to just ensure that this view only accepts GET/POST?

Comment (ryanmrubin) [inline cas_consumer/views.py]: Looking at this with sleep and coffee, yes. Coming up.

**Comment by eykd** [comment]
:+1:

**Pull request merged**